### PR TITLE
mega target group v2.7.2

### PR DESCRIPTION
**Pull request by ylee-figma**
- This effectively passes down the CIDR blocks as a flag and filters the list of targets that are returned from aws.
- parse cidrs at main func
- filter out epAddr in endpoint resolver
- instead of reading cidrs from the flag, we can incrementally onboard new services

### CODE REVIEW

File: pkg/targetgroupbinding/resource_manager.go [hunk 1]
```diff
@@ -539,3 +549,17 @@ func isELBV2TargetGroupARNInvalidError(err error) bool {
 	}
 	return false
 }
+
+func parseCIDR(tgb *elbv2api.TargetGroupBinding) ([]netip.Prefix, error) {
+	var cidrs []netip.Prefix
+	// only triggered if the object has an annotation named `annotation`
+	if annotation := tgb.Annotations["cidrs"]; annotation != "" {
```
Comment (dlmather) [inline pkg/targetgroupbinding/resource_manager.go]: update to something a lil more saucy

**Review by dlmather — APPROVED** [review]

File: pkg/targetgroupbinding/resource_manager.go (hunk 1 above)
Comment (ylee-figma) [reply pkg/targetgroupbinding/resource_manager.go]: `mega-target-group-cidrs` no one can stop me

Comment (ylee-figma) [reply pkg/targetgroupbinding/resource_manager.go]: I just renamed it to filter-cidrs

**Comment by ylee-figma** [comment]
original implementation: https://github.com/dlmather/aws-load-balancer-controller/pull/1

**Pull request merged**